src/compas/geometry/vector.py

```diff
@@ -190,6 +200,9 @@ def __ipow__(self, n):
         self.z **= n
         return self
 
+    def __rmul__(self, n):
```

Comment:
no `__rsub__`, `__radd__`? or do these not exist?

Reply:
they do, let me try this too...